client/checkout/woopay/direct-checkout/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import WooPayDirectCheckout from 'wcpay/checkout/woopay/direct-checkout/woopay-direct-checkout';
+
+window.addEventListener( 'load', async () => {
+	if ( ! WooPayDirectCheckout.isWooPayEnabled() ) {
+		return;
+	}
+
+	WooPayDirectCheckout.init();
+
+	const checkoutElements = WooPayDirectCheckout.getCheckoutRedirectElements();
+	const isThirdPartyCookieEnabled = await WooPayDirectCheckout.isWooPayThirdPartyCookiesEnabled();
```

Comment:
We discussed on our call about creating an issue to explore alternative ways to determine third party cookie status. We also discussed that once we add things like the mini-cart to the selectors, this could run on every page load so we should just run this once and store on a session level cookie for example.